cli/cmd/tcm.go

```diff
@@ -2,73 +2,103 @@ package cmd
 
 import (
 	"errors"
+	"fmt"
+	"log"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"time"
 
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/jedib0t/go-pretty/v6/text"
 	"github.com/spf13/cobra"
 	"github.com/tarantool/tt/cli/cmdcontext"
-	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/process_utils"
 	tcmCmd "github.com/tarantool/tt/cli/tcm"
-	"github.com/tarantool/tt/cli/util"
+	libwatchdog "github.com/tarantool/tt/lib/watchdog"
 )
 
 var tcmCtx = tcmCmd.TcmCtx{}
 
+const (
+	tcmPidFile      = "tcmPidFile.pid"
+	watchdogPidFile = "watchdogPidFile.pid"
```

Comment:
We don't need the suffix since we have a file extension that told us that it is a pid file.
```suggestion
	tcmPidFile      = "tcm.pid"
	watchdogPidFile = "watchdog.pid"
```

Reply:
done